SPARCore/GADArticle.m

```diff
@@ -12,35 +11,74 @@ @implementation GADArticle
 static NSString *const API_PUBLICATION_ID = @"publication";
 static NSString *const API_TITLE = @"title";
 
-static NSString *const API_HOSTNAME = @"https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/";
-static NSString *const API_PREFIX = @"devstable";
+static NSString *const API_ITMES = @"items";
+static NSString *const API_NEXT_PAGE_TOKEN = @"nextPageToken";
+
 static NSString *const API_PUBLICATION_PATH = @"publications";
 static NSString *const API_ARTICLE_PATH = @"articles";
 
-+ (void) articlesFromPublication: (NSString *)publicationId
-                                        completionHandler:(void(^_Nonnull)(NSArray<GADArticle *>
-                                                                           *_Nullable articles,
-                                                            NSError *_Nullable error))completion {
-    NSURL *queryURL = [GADArticle urlForArticlesFromPublication:publicationId];
++ (void) articlesForPublicationId: (NSString *)publicationId
```

Comment:
The significance of the path is different though, right? We're comparing `/articles/{id}` to `/publications/{id}/articles`, right? Though they're the same word, @AlexanderOtavka could change the API such that one could be different but not the other. Because of this, we should have a separate constant in `GADPublication` that relates to "the path which fetches the articles for this publication".